config/services/payum.yaml

```diff
@@ -2,28 +2,38 @@ services:
     _defaults:
         public: true
 
-    smile_hipay_sylius_plugin.request.get_http_request:
-        class: Payum\Core\Request\GetHttpRequest
-        
-    Smile\HipaySyliusPlugin\Payum\Action\StatusAction:
-        class: Smile\HipaySyliusPlugin\Payum\Action\StatusAction
+#    smile_hipay_sylius_plugin.request.get_http_request:
+#        class: Payum\Core\Request\GetHttpRequest
+
+    Smile\HipaySyliusPlugin\Payum\Extension\ClearPaymentContextExtension:
+        class: Smile\HipaySyliusPlugin\Payum\Extension\ClearPaymentContextExtension
         arguments:
-            - "@smile_hipay_sylius_plugin.request.get_http_request"
             - '@Smile\HipaySyliusPlugin\Context\PaymentContext'
-            - '@Smile\HipaySyliusPlugin\Security\HipaySignatureVerification'
```

Comment:
Please don't forget to re-implement the Hipay Signature Verification service

Reply:
Done